spec/support/csv/Tagged_assessment_invalid_values.csv

```diff
@@ -1,5 +1,5 @@
 Measurement_date,SHL_case_number,Spawning_date,Tag,From_Growout_Rack,From_Growout_Column,From_Growout_Trough,To_Growout_Rack,To_Growout_Column,To_Growout_Trough,Length,Gonad_Score,Predicted_Sex,Notes
 12/17/18,SF16-9A,3/2/16,Lav_028,3,A,1,1,what's,the,good,word?,M,
 12/17/18,SF16-9A,3/2/16,Lav_057,3,A,to hell with georgia,1,A,2,49,1,M,
-12/17/18,SF16-9A,3/2/16,Lav_029,3,A,1,1,A,2,53.5,1,F,
-12/17/18,SF16-9A,3/2/16,Lav_078,3,A,1,1,A,2,50,1,F,
+12/17/18,SF16-9A,3/2/16,Lav_029,3,A,1,1,how,'bout,them,dawgs?,F,
+12/17/18,SF16-9A,3/2/16,Lav_078,3,A,1,1,A,piss,on,them!,F,

```

Comment:
Is this entire test suite a football reference? Let the record show I don't condone any sportsball teams bashing.
But if it's referring to just dogs I'll let it slide.
🐱 > 🐶